flowzone.yml

```diff
@@ -2270,6 +2259,19 @@ jobs:
           echo "${result}" >> $GITHUB_OUTPUT
           echo "${DELIMITER}" >> $GITHUB_OUTPUT
 
+      # if auto_merge is enabled at the repository level, configure PR auto-merge
+      - name: Enable PR auto-merge
+        if: |
+          github.event.pull_request.draft == false &&
+          needs.event_types.outputs.do_draft == 'true' &&
+          inputs.repo_config == true
```

Comment:
```suggestion
```
Might as well delete this check since we are effectively making this change if branch protection is true and we are on an active PR. Does not depend on repo config anymore.

Reply:
but if we remove this conditional, then auto-merge will be enabled by default with the next run..